components/mpas-ocean/src/shared/mpas_ocn_vel_self_attraction_loading.F

```diff
@@ -170,8 +173,11 @@ subroutine ocn_compute_self_attraction_loading(domain, forcingPool, dminfo, ssh,
             return
         endif
 
+        rho0gInv = 1.0_RKIND / rho_sw / gravity
+
         do iCell = 1,nCellsAll
-          sshSmoothed(iCell) = coastalSmoothingFactor(iCell)*ssh(iCell)
+          sshSmoothed(iCell) = coastalSmoothingFactor(iCell) * ( &
+              ssh(iCell) + rho0gInv * surfacePressure(iCell) )
         enddo
```

Comment:
Maybe add a comment above this to clarify that the surface pressure contribution is being added to account for the full bottom pressure (in a barotropic sense) and that that this is being smoothed to avoid issues with dry cells, etc. I think `sshSmoothed` variable name should be updated also.